vmdb/app/models/container_node.rb

```diff
@@ -3,5 +3,6 @@ class ContainerNode < ActiveRecord::Base
   # :name, :uid, :creation_timestamp, :resource_version
   belongs_to :ext_management_system, :foreign_key => "ems_id"
   has_many   :container_groups
+  has_one    :hardware, :dependent => :destroy
```

Comment:
@Fryguy @blomquisg I suppose you're talking about OperatingSystem (not ComputerSystem). Correct?